src/Loop.php

```diff
@@ -124,7 +120,8 @@ public static function get()
      */
     public static function stop()
     {
-        self::get()->stop();
+        $driver = self::$driver ?: self::get();
```

Comment:
You can as well just directly call `self::createDriver()` instead of `self::get()`.